packages/react-dom/src/client/DOMPropertyOperations.js

```diff
@@ -172,3 +180,17 @@ export function setValueForProperty(
     }
   }
 }
+
+// Only used in DEV, stringifies a value and Warns if it took too long
+const stringifyWithPerformanceWarning = value => {
+  const stringifyStart = Date.now();
+  let attributeValue = (value: any).toString();
```

Comment:
Use `const` over `let`.

Reply:
Please use `'' + value` instead of `toString` here as well.